src/kafka.rs

```diff
@@ -110,15 +111,14 @@ async fn handle_message(
     message: &BorrowedMessage<'_>,
 ) -> Result<(), Error> {
     match decode_message(decoder, message).await? {
-        InputEvent::DatasetEvent(event) => {
-            let key = event.fdk_id.clone();
-            tracing::info!("dataset event with id {} decoded", key);
+        InputEvent::HarvestEvent(event) => {
+            return post_event_graph_to_diff_store(event).await;
```

Comment:
Trenger nok ikke return her:
```suggestion
            post_event_graph_to_diff_store(event).await
```

Reply:
fix'nd't